### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by derwaldgeist** [studiointeract/accounts-ui#142]
This package looks interesting. But since major issues like React Router v4 support haven't been resolved since months, I am wondering if it's still safe to use it? Looks kinda abandoned to me.

**Comment by njbuch** [comment]
> This package looks interesting. But since major issues like React Router v4 support haven't been resolved since months, I am wondering if it's still safe to use it? Looks kinda abandoned to me.

Agree, and thats a shame...

**Comment by thebarty** [comment]
@derwaldgeist Are you using this package? There is another on… https://github.com/royGil/accounts-react

**Comment by derwaldgeist** [comment]
Nope, I am not using this, because it does not seem to be maintained anymore. Now using the one you mentioned.

**Comment by filipenevola** [comment]
Hi, you should use this https://atmospherejs.com/epotek/accounts-ui
